content/docs/ops/setup/injection/index.md

```diff
@@ -40,6 +45,16 @@ these three variables.
 This includes an injected sidecar when it wasn't expected and a lack
 of injected sidecar when it was.
 
+1. Ensure your pod is not in the `kube-system` or `kube-public` namespace.
+   Automatic sidecar injection cannot be used in these namespaces.
```

Comment:
Can you be more clear about the limitation, e.g., what would happen if you turn on an annotation? Does Istio ignore it, or will very bad things happen :-) Saying something like `Automatic sidecar injection configuration is ignored in these namespaces.` would be less mysterious IMO.